src/core/init.js

```diff
@@ -14,6 +14,15 @@ import { initialize as initTranslator } from './internationalization';
  * @return {Undefined}
  */
 const _globalInit = () => {
+  // Could have been any property defined within the p5 constructor.
+  // If that property is already a part of the global object,
+  // this code has already run before, likely due to a duplicate import
+  if (typeof window._setupDone !== 'undefined') {
+    throw new Error(
```

Comment:
I'm not sure throwing an error is necessary. We just need for it to not initialize in the case that p5.js is already loaded. A warning and an early `return` should be more appropriate I think.

Reply:
Okay. But I felt a warning would be apt if we didn't cut the initialisation midway, and that way warn the user about running with dual initialisation. Hence I used an error here. But I am wasn't sure.

Reply:
Either way is fine, I just feel an error is a bit too loud when there is mitigation (stopping second initialization) and so the user doesn't necessarily need to do anything else to continue. Error feels more like something is wrong and it must be fixed before continuing. Just what I feel though.

Reply:
Should I change this to a warning and an early return then?